cocos/game/game.ts

```diff
@@ -291,6 +289,26 @@ export class Game extends EventTarget {
      */
     public static readonly EVENT_RESTART = 'game_on_restart';
 
+    /**
+     * @en Triggered when the user triggers the game pause.<br>
+     * @zh 用户触发游戏暂停时触发该事件。<br>
+     * @example
+     * ```ts
+     * import { game, audioEngine } from 'cc';
```

Comment:
```suggestion
     * import { game } from 'cc';
```

we don't provide audioEngine